app/controllers/admins_controller.rb

```diff
@@ -13,6 +13,13 @@ def approve
         @item.save
         redirect_to admins_path
     end
+    
+    def approve_p
```

Comment:
make this method's name more descriptive what does the p stand for?

Reply:
Changed the names to approve_item and approve_procedure.